### PR TITLE
php 7.2.10

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This also bumps devel, not sure if that should be in the commit message.

### CODE REVIEW

File: Formula/php.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Php < Formula
   desc "General-purpose scripting language"
   homepage "https://secure.php.net/"
-  url "https://php.net/get/php-7.2.9.tar.xz/from/this/mirror"
-  sha256 "3585c1222e00494efee4f5a65a8e03a1e6eca3dfb834814236ee7f02c5248ae0"
+  url "https://php.net/get/php-7.2.10.tar.xz/from/this/mirror"
+  sha256 "01c2154a3a8e3c0818acbdbc1a956832c828a0380ce6d1d14fea495ea21804f0"
   revision 2
```
Comment (fxcoudert) [inline Formula/php.rb]: `revision` goes away

**Comment by fxcoudert** [comment]
I think stable trumps devel in the commit message :)

**Comment by SMillerDev** [comment]
I have more than one line to put a message.
Anyway, I'll update it in an hour or so.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
`brew pull` said message needed to be `php 7.2.10, 7.3.0RC1 (devel)`

**Comment by SMillerDev** [comment]
If brew pull says so it must be true

**Pull request closed without merging**